easybuild/easyconfigs/f/FlexiBLAS/FlexiBLAS-3.0.4-GCC-10.2.0.eb

```diff
@@ -0,0 +1,23 @@
+name = 'FlexiBLAS'
+version = '3.0.4'
+
+homepage = 'https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release'
+description = """FlexiBLAS is a wrapper library that enables the exchange of the BLAS and LAPACK implementation
+used by a program without recompiling or relinking it."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+source_urls = ['https://csc.mpi-magdeburg.mpg.de/mpcsc/software/flexiblas/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['50a88f2e88994dda91b2a2621850afd9654b3b84820e737e335687a46751be5c']
+
+builddependencies = [('CMake', '3.18.4')]
+
+# note: first listed library will be used as default by FlexiBLAS,
+# unless otherwise specified via -DFLEXIBLAS_DEFAULT in configopts
```

Comment:
this is currently not checked in the easyblock, so if you set -DFLEXIBLAS_DEFAULT in configopts, the easyblock will override it.

Reply:
Current easyblock is making sure it doesn't overwrite configure options specified via `configopts` (and I checked that it works)